azure-pipelines.yml

```diff
@@ -31,12 +31,15 @@ jobs:
                 architecture: 'x64'
 
           - script: |
-                pip install "spacy>=3.1.0,<3.2.0"
+                pip install "spacy>=3.2.0,<3.3.0"
```

Comment:
Do you think there will be something breaking in 3.3? Considering that release is near, it might be nice to make sure it's supported too?

Reply:
Sure, but maybe in a separate PR after v3.3 is released?